[vividus] Allow handling of empty tables to CARTESIAN_PRODUCT transformer according to mathematical logic (#3357)
vividus-framework/vividus: docs/modules/commons/pages/table-transformers.adoc

====
* headers of input tables must be different
* number of input tables must be more than 1

> * if one table is empty - the transformed table will also be empty

Reviewer: ```suggestion
* if any of tables is empty - the resulting table will also be empty
```
